pages/over-ons.tsx

```diff
@@ -1,11 +1,75 @@
-import { Container, Layout } from '@components'
+import React from 'react'
+
+import { Box, Heading, Stack, Text } from '@chakra-ui/react'
+import { GetStaticProps } from 'next'
+import { useTranslation } from 'next-i18next'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+import { Container, Hero, Layout } from '@components'
 
 const OverOns = (): JSX.Element => {
+  const { t } = useTranslation()
   return (
     <Layout>
```

Comment:
Can you move `Hero` outside the container. 

For transparent Header background  please add the following prop to `Layout`.

```suggestion
    <Layout scrollHeight={100}>
```